addons/mrp_subcontracting_account/models/stock_move.py

```diff
@@ -45,3 +45,13 @@ def _generate_valuation_lines_data(self, partner_id, qty, debit_value, credit_va
                 'account_id': component_cost_account.id,
             }
         return rslt
+
+    def _get_dest_account(self, account_data):
+        if self.raw_material_production_id.subcontractor_id:
```

Comment:
I would prefere fallbacking on original `_get_dest_account` method instead of rewriting the same code. 
Could you instead make the method `_is_production_consumed` return `False` in case of subcontracting ?
https://github.com/odoo/odoo/blob/6ea8ca21ae3359f0481723fff2b54af7c0b55f66/addons/mrp_account/models/stock_move.py#L37-L40

Reply:
This code is not available on 16.2, I can do it for 16.3 and onwards